### ISSUE TITLE
"Sponsored" Advertisements Showing Up in Revanced

### ISSUE THREAD

**Issue by TheSubMatrix** [ReVanced/revanced-patches#3412]
### Bug description

On the latest recommended patch of YouTube (18.45.43) patched on the latest release version of revanced manager (1.17.3) with root patching, I am beginning to see sponsored advertisements popping up on the home page and in my recommended. The advertisements do not appear to being correctly blocked by the existing patches. 
![Screenshot_2023-12-12-23-50-54-28_f9ee0578fe1cc94de7482bd41accb329](https://github.com/ReVanced/revanced-patches/assets/25988883/857b84c2-38f0-4af0-a621-7cd4c128e25e)

The issues began sometime earlier today on an older patched version of vanced. I updated to see if perhaps the error was resolved on a future patch and this did not appear to fix the issue.

Below are the logcat files from the bug.
[13_12-00-47-58_889.log](https://github.com/ReVanced/revanced-patches/files/13656760/13_12-00-47-58_889.log)

Below is my patch list
[selected_patches_2023-12-13_00_15_11.json](https://github.com/ReVanced/revanced-patches/files/13656764/selected_patches_2023-12-13_00_15_11.json)

### Error logs

```shell
Listed in the above description
```

### Acknowledgements

- [X] This issue is not a duplicate of an existing bug report.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.

**Comment by johnconner122** [comment]
Try entering `video_display_carousel_button_group_layout` in custom filter.
